Gruntfile.js

```diff
@@ -240,7 +314,7 @@ module.exports = function (grunt) {
                 files: [{
                     expand: true,
                     src: ["src/**/css/stylus/*.styl"],
-                    ext: ".css",
+                    ext: "<% buildSettings.compress ? print('.min.css') : print('.css') %>",
```

Comment:
what does the `print` method do in this case? for example would it be possible to just write `"<% buildSettings.compress ? '.min.css' : '.css' %>"`